src/js_native_api_v8.cc

```diff
@@ -1706,6 +1706,30 @@ napi_status NAPI_CDECL node_api_create_external_string_utf16(
       });
 }
 
+napi_status node_api_create_property_key_latin1(napi_env env,
+                                                const char* str,
+                                                size_t length,
+                                                napi_value* result) {
+  return v8impl::NewString(env, str, length, result, [&](v8::Isolate* isolate) {
+    return v8::String::NewFromOneByte(isolate,
+                                      reinterpret_cast<const uint8_t*>(str),
+                                      v8::NewStringType::kInternalized,
```

Comment:
Why not kNormal?

Reply:
hello thank you very much for your comment @anonrig 

I thought this would be better in terms of performance
Memory Savingand such as Fast Comparisons

Reply:
If these values can be user controlled, then `kNormal` is likely better here.

Reply:
I applied this, thank you very much

Reply:
@jasnell , could you please provide more details on the issues with the Internalized strings.
The goal of these new APIs is to create strings to be used as property keys. The internalized strings must be more efficient because otherwise V8 must internalize Normal string every single time.